src/block_cache/block_chunkcache.c

```diff
@@ -261,6 +261,37 @@ __chunkcache_eviction_thread(void *arg)
  * __wt_chunkcache_get --
  *     If the cache has the data at the given size and offset, copy it into the supplied buffer.
  *     Otherwise, read and cache the chunks containing the requested data.
+ *
+ * The locks protect two data structures: (1) the buckets of the hash table that keeps track of the
+ *     allocated chunks and the (2) LRU list of chunks, used by eviction. Each bucket in the hash
+ *     table is protected by a single spinlock. The LRU list is also protected by a single lock.
+ *
+ * Lock ordering: * If we ever need to hold a bucket lock and the LRU list lock simultaneously, we
+ *     always acquire the bucket lock first and the LRU lock second. * In the current
```

Comment:
In this paragraph there are two `*` in the text, probably leftover from reformatting.

Perhaps related, the formatting style in this comment is not standard.  Instead of
```
 * First line
 *      second line
 *      third line
```
It should be
```
 * First line
 * second line, etc.
```
If you do it the second way and run `dist/clang-format -F`, your text will be formatted.  The formatter skips over indented comment lines, so it's a good way to insert something that shouldn't be touched, but not really for regular text.